### ISSUE TITLE
Add support `--type tcpdump`

### ISSUE THREAD

**Issue by yoku0825** [yoku0825/anemoeater#8]
Suppose syntax

```
tcpdump -s 65535 -x -nn -q -tttt -i any -c 1000 port 3306 > mysql.tcp.txt
anemoeater --tcpdump mysql.tcp.txt
```

https://www.percona.com/doc/percona-toolkit/LATEST/pt-query-digest.html#cmdoption-pt-query-digest-type

**Comment by yoku0825** [comment]
#9 proposes for this enhancement.
Thank you @yokogawa-k !

**Comment by yoku0825** [comment]
Closed by #9
